### PR TITLE
chore: [PL-20548]: update ui-core version for Fix for Placeholder in tag component

### PR DESCRIPTION
**Pull request by mayankVermaHarness**
##### Summary:

Corresponding UI-Core PR: https://github.com/harness/uicore/pull/579

##### Jira Links:

https://harness.atlassian.net/browse/PL-20548

##### Screenshots:

**Before:**
image

**After:**
image

**Before:**
image

**After:**
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by avinashmadhwani02 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "@harness/monaco-yaml": "^1.0.0",
     "@harness/ng-tooltip": "^1.30.68",
     "@harness/telemetry": "^1.0.37",
-    "@harness/uicore": "^2.2.0",
+    "@harness/uicore": "^2.3.0",
```
Comment (avinashmadhwani02) [inline package.json:92]: let's use exact version, remove ^

**Review by reetika04 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (abhinavrastogi-harness) [reply package.json:92]: please dont change this. it can have dangerous side effects with microfrontends.

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]
@mayankVermaHarness i think you manually changed in package.json?
use `yarn upgrade` instead so that yarn.lock also updates.
otherwise uicore will not actually get updated.

**Comment by mayankVermaHarness** [comment]
retrigger jest

**Comment by mayankVermaHarness** [comment]
retrigger jest

**Pull request closed without merging**